app/src/main/java/com/teamltt/carcare/activity/AlertActivity.java

```diff
@@ -40,23 +40,9 @@ public boolean onCreateOptionsMenu(Menu menu) {
     }
 
     public boolean onOptionsItemSelected(MenuItem item) {
-        Intent intent;
         switch (item.getItemId()) {
-            case (R.id.action_car_info):
-                intent = new Intent(this, CarInfoActivity.class);
-                startActivity(intent);
-                break;
-            case (R.id.action_trips):
-                intent = new Intent(this, TripsActivity.class);
-                startActivity(intent);
-                break;
-            case (R.id.action_dynamic):
-                intent = new Intent(this, DynamicActivity.class);
-                startActivity(intent);
-                break;
-            case (R.id.action_reminder):
-                intent = new Intent(this, ReminderActivity.class);
-                startActivity(intent);
```

Comment:
These classes should be accessible through the drawer. I talked to Jack and he approved this.